docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -98,6 +99,9 @@ for instance the MediaBundle defines the method as::
         }
     }
 
+.. note::
+    To get the provider from your ``Admin`` class, use this:
+    ``$provider = $this->getConfigurationPool()->getContainer()->get($object->getProviderName());``
```

Comment:
This chaining of methods is a bad practice. Can you please add a different example with DI

Reply:
Done. Thanks for advise.